### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/cred.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4872]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4872|
|Type|enhancement|
|Created|2011-02-11 05:35:50Z|
|Branch|https://github.com/twisted/twisted/tree/cred-sphinx-4872|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the cred howto

<details><summary>Searchable metadata</summary>

```
trac-id__4872 4872
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_cred_sphinx_4872 branches/cred-sphinx-4872
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297402550000000 1297402550000000
changetime__1300076054000000 1300076054000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30676]) Branching to 'cred-sphinx-4872'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [30677]) Merge the cred parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs #4568
refs [#4872](https://github.com/twisted/twisted/issues/4872)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)<a name="note_5"></a>|@moshez set owner to @exarkun|
|-|-|

There is a move of newline from before "is" (in "the mind is") to after it, and a couple of trailing spaces. Why is that?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun removed owner|
|-|-|

To fix the generated ReST markup.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_9"></a>|@lvh commented|
|-|-|

There are three changes in this diff. The first two produce [broken output](http://buildbot.twistedmatrix.com/builds/sphinx-html/199-15779/projects/core/howto/cred.html), the last one is fine.

 1. put "is" on the next line, problem is missing space, resulting in None not being monospace (generated reST: ```Typically you only pass in one interface, and the mind is``None`````)
 2. putting `*interfaces` in `<code>` tags, converter doesn't add backticks, Sphinx somehow manages to interpret the result as a link (???) (generated reST: ```which was one of the interfaces passed in the *interfaces tuple```)
 3. something about myapp_plugins.py, which works fine


This is impossible to fix in the conversion script, so this Sphinx file requires manual intervention. Neither change is particularly hard: one space and two pairs of backticks.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_10"></a>|@lvh commented|
|-|-|

We're merging this anyway. See [#4940](https://github.com/twisted/twisted/issues/4940) for actually fixing the problems in the last comment.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_11"></a>|@lvh set status to `closed`|
|-|-|

(In [31003]) Merge cred.xhtml changes from cred-sphinx-4872.


Author: screwtape, jdb
Reviewer: lvh
Fixes: [#4872](https://github.com/twisted/twisted/issues/4872)
  
"Makes Sphinx looks gooder." -- JP

There are problems left, for which there is [#4940](https://github.com/twisted/twisted/issues/4940).
